dotCMS/src/curl-test/Add-Remove Contentlet from a Page.postman_collection.json

```diff
@@ -0,0 +1,508 @@
+								}
+							]
+						}
+					},
+					"response": []
+				},
+				{
+					"name": "Render again after Add/remove",
+					"event": [
+						{
+							"listen": "test",
+							"script": {
+								"exec": [
+									"pm.test(\"Variants with correct weight\", function () {",
+									"    pm.response.to.have.status(200);",
+									"   ",
+									"   var html = pm.response.json().entity.page.rendered;",
+									"    pm.expect(html).to.contains('content_1');",
+									"    pm.expect(html).to.contains('content_4');",
+									"",
+									"    pm.expect(html).not.to.contains('content_2');",
+									"    pm.expect(html).not.to.contains('content_3');",
+									"});"
+								],
+								"type": "text/javascript"
+							}
+						}
+					],
+					"request": {
+						"auth": {
+							"type": "noauth"
+						},
+						"method": "GET",
+						"header": [],
+						"url": {
+							"raw": "{{serverURL}}/api/v1/page/render/page_test_add_remove_contentlet?language_id=1&variantName=dotexperiment-{{experimentShortId}}-variant-1",
+							"host": [
+								"{{serverURL}}"
+							],
+							"path": [
+								"api",
+								"v1",
+								"page",
+								"render",
+								"page_test_add_remove_contentlet"
+							],
+							"query": [
+								{
+									"key": "language_id",
+									"value": "1"
+								},
+								{
+									"key": "variantName",
+									"value": "dotexperiment-{{experimentShortId}}-variant-1"
+								}
+							]
+						}
+					},
+					"response": []
+				},
+				{
+					"name": "Render again after Add/remove DEFAULT variant",
+					"event": [
+						{
+							"listen": "test",
+							"script": {
+								"exec": [
+									"pm.test(\"Variants with correct weight\", function () {",
+									"    pm.response.to.have.status(200);",
+									"   ",
+									"    var html = pm.response.json().entity.page.rendered;",
+									"    pm.expect(html).to.contains('content_1');",
+									"    pm.expect(html).to.contains('content_2');",
+									"    pm.expect(html).to.contains('content_3');",
+									"",
+									"    pm.expect(html).not.to.contains('content_4');",
+									"});"
+								],
+								"type": "text/javascript"
+							}
+						}
+					],
+					"request": {
+						"auth": {
+							"type": "noauth"
+						},
+						"method": "GET",
+						"header": [],
+						"url": {
+							"raw": "{{serverURL}}/api/v1/page/render/page_test_add_remove_contentlet?language_id=1",
+							"host": [
+								"{{serverURL}}"
+							],
+							"path": [
+								"api",
+								"v1",
+								"page",
+								"render",
+								"page_test_add_remove_contentlet"
+							],
+							"query": [
+								{
+									"key": "language_id",
+									"value": "1"
+								}
+							]
+						}
+					},
+					"response": []
+				}
+			]
+		}
+	],
+	"variable": [
+		{
+			"key": "experimentId",
+			"value": ""
+		},
+		{
+			"key": "experimentShortId",
+			"value": ""
+		}
+	]
+}
```

Comment:
Being honest the bundles have been a bit complicated, I would prefer to use pure postman test if possible

Reply:
done https://github.com/dotCMS/core/pull/23258/commits/41c0be63fe5abada0ae520613291c2ac5a0a080a